tools/toc.sh

```diff
@@ -9,7 +9,7 @@
 let ERROR=0
 
 # Find all markdown files
-for f in $(find . | grep .md); do
+for f in $(find doc/ boards/ kernel/ arch/ capsules/ chips/ userland/ -name "*.md"); do
```

Comment:
Yes, well this whole script is kind of problematic in that sense... Piping to grep isn't terrible except it's a large number of files, when `find` is already doing the work of filtering anyway.

We can use `git ls-tree --name--only -r HEAD **/*.md` to get the best of both worlds, except it excludes the working tree (which is good for CI but bad for development)